src/DotNetCore.CAP/Internal/ISubscribeInvoker.Default.cs

```diff
@@ -36,7 +36,7 @@ public async Task<ConsumerExecutedResult> InvokeAsync(ConsumerContext context, C
             cancellationToken.ThrowIfCancellationRequested();
 
             var methodInfo = context.ConsumerDescriptor.MethodInfo;
-            var reflectedType = methodInfo.ReflectedType.Name;
+            var reflectedType = methodInfo.ReflectedType.TypeHandle.Value;
```

Comment:
If we use `ReflectedType.TypeHandle.Value`, as described in #908, do we still need `"{methodInfo.Module.Name}_{reflectedType}_{methodInfo.MetadataToken}"` to ensure that the key is unique?

Reply:
Hey got slammed with work this week. I was trying to keep the PRs separate so I was planning to do this once I had some test coverage in place in a subsequent PR. happy to amalgamate tho if that's easier.